Sources/MapboxMaps/Gestures/GestureType.swift

```diff
@@ -1,24 +1,19 @@
-import Foundation
-
 public enum GestureType: Hashable, CaseIterable {
     /// The pan gesture
     case pan
 
-    /// The double tap to zoom in gesture
-    case doubleTapToZoomIn
-
-    /// The double tap to zoom out gesture
-    case doubleTapToZoomOut
-
     /// The pinch gesture
     case pinch
 
-    /// The rotate gesture
-    case rotate
+    /// The pitch gesture
+    case pitch
+
+    /// The double tap to zoom in gesture
+    case doubleTapToZoomIn
+
+    /// The double touch to zoom out gesture
+    case doubleTouchToZoomOut
```

Comment:
nit: Let's order these alphabetically

Reply:
Can we debate this in a separate ticket?